python/lsst/ts/Dome/mock_controller.py

```diff
@@ -243,6 +248,9 @@ async def stop_el(self):
         self.log.info("Received command 'stopEl'")
         self.el_motion = "Stopped"
 
+    async def config(self, **kwargs):
+        self.log.info(f"Received command 'moveAz' with arguments {kwargs}")
```

Comment:
Please update the string. If you want to print such a message for every command method then please consider logging the message in the code that calls the method -- one message would handle all commands.

Reply:
This is a copy and paste error and I will fix it.